components/03-modules/gallery/gallery.hbs

```diff
@@ -0,0 +1,86 @@
+<div class="
+         gallery {{ class }}
+         {{#if typeHorizontal }}
+             gallery--horizontal
+         {{else}}
+             gallery--vertical
+         {{/if}}
+     "
+     {{{ attributes }}}
+>
+    <div class="
+             gallery__stage
+             {{ stage.class }}
+             {{#if typeHorizontal }}
+                 gallery__stage--horizontal
+             {{else}}
+                 gallery__stage--vertical
+             {{/if}}
+         "
+    >
+        <div class="gallery__arr gallery__arr--prev">
+        </div>
+        <div class="gallery__shaft">
+            <div class="gallery__frame">
+                {{#if productLabel }}
+                <span class="
+                          gallery__product-label
+                          gallery__product--{{productLabel.type}}
+                      "
+                >
+                    {{ productLabel.text }}
+                </span>
+                {{/if}}
+                {{ render '@image' imageMain }}
+            </div>
+        </div>
+        <div class="gallery__arr gallery__arr--next">
+        </div>
+    </div>
+    <div class="gallery__nav
+                {{#if typeHorizontal }}
+                    gallery__nav--horizontal
+                {{else}}
+                    gallery__nav--vertical
+                {{/if}}
+                {{ nav.class }}
+         "
+   >
+        <div class="gallery__thumb-arr">
+            <svg class="
+                     gallery__icon-arrow
+                     {{#if typeHorizontal }}
+                         gallery__icon-arrow--horizontal
+                     {{else}}
+                         gallery__icon-arrow--vertical
+                     {{/if}}
+            ">
+                <use xlink:href="{{ svg 'arrow-small-up' }}"></use>
+            </svg>
+        </div>
+        {{#each thumbs }}
+            <div class="
+                     gallery__thumb
+                     {{#if imageThumb.active}}
+                         gallery__thumb--active
+                     {{/if}}
+                 "
+            >
+                {{ render '@image' imageThumb }}
+            </div>
+        {{/each}}
+        <div class="gallery__thumb-arr">
```

Comment:
same question: in Magento it's `a` tag or `button` or just `div`?

Reply:
div so I put div and it's just placeholder so wouldn't make it perfect as Iglo said it's just for preview

Reply:
ok